### DISCUSSION TITLE
Group blogs into collections - How ?

### DISCUSSION THREAD

**Discussion by ThomasFr42** [TumblThreeApp/TumblThree#296] [Q&A (Question & Answer)]
Ther is a new feature:
Group blogs into collections

but I found no documentation how to do that ?
Is this explained in the wiki ?

**Comment by thomas694** [comment]
I haven't had time yet to write an article about it. You can only look at the originating [issue #170](https://github.com/TumblThreeApp/TumblThree/issues/170) and try it out yourself.

Everyone who would like to write a short tutorial on how to use the new functionality is very welcome.
